plugin-gradle/src/main/java/com/diffplug/gradle/spotless/GitRatchetGradle.java

```diff
@@ -34,4 +38,9 @@ protected File getDir(Project project) {
 	protected @Nullable Project getParent(Project project) {
 		return project.getParent();
 	}
+
+	@Override
+	public void onFinish(FinishEvent finishEvent) {
+		// NOOP
```

Comment:
Just want to confirm that we don't need to call `close()` here? `GitRatchet implements AutoCloseable`, so if Gradle is promising to call `close()` then I guess we're good, but it's odd that there's also an `onFinish()` which seems like it would get called at the same time as `close()`?

Reply:
`OperationCompletionListener.onFinish` is called on each "task finish" event.
`AutoCloseable.close` is called at the end of the build.

My understanding is that we are good.